recipes/libcurl/all/conanfile.py

```diff
@@ -191,9 +197,20 @@ def _get_configure_command_args(self):
         if tools.cross_building(self.settings):
             if self.settings.os == "Linux" and "arm" in self.settings.arch:
                 params.append('--host=%s' % self._get_linux_arm_host())
+            elif self.settings.os == "iOS": 
+                params.append("--enable-threaded-resolver") 
+                params.append("--disable-verbose")
+                # if there is anything special for arm or simulator, add it here
+                if "arm" in self.settings.arch:
```

Comment:
I don't think it's nice to leave code blocks doing nothing right now

Reply:
maybe for documentation purpose it is OK ?